src/main/sensors/gyro.c

```diff
@@ -681,7 +682,7 @@ STATIC_UNIT_TESTED void performGyroCalibration(gyroSensor_t *gyroSensor, uint8_t
                 gyroSetCalibrationCycles(gyroSensor);
                 return;
             }
-            gyroSensor->gyroDev.gyroZero[axis] = (gyroSensor->calibration.sum[axis] + (gyroCalculateCalibratingCycles() / 2)) / gyroCalculateCalibratingCycles();
```

Comment:
This is needed to avoid rounding errors. Keep in mind that gyroZero is int, not float. So you need to add gyroCalculateCalibratingCycles() / 2 before rounding to int.